packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -99,11 +103,11 @@ const AppContent = (): JSX.Element => {
     )
   }
 
-  if (connectionState === ConnectionState.NOT_EOA) {
+  if (walletType === WalletType.UNSUPPORTED_CONTRACT_WALLET) {
+    // TODO: better alert text / keep it?
```

Comment:
might need to change, don't like the `This shouldn't happen` part as it can easily happen if the L1 and L2 safe have different owner